shared-libs/transitions/package.json

```diff
@@ -26,8 +26,10 @@
     "async": "^3.2.3",
     "bikram-sambat": "^1.7.0",
     "bikram-sambat-bootstrap": "^1.5.0",
+    "cron-validator": "^1.3.1",
```

Comment:
@dianabarsan The package-lock.json isn't updated after the install, it keeps routing to the root package-lock.json.

Reply:
Uh, you're right! It's because of npm workspaces. I've been trying to find some way to get npm to install the package in the correct place, but no luck yet. We're going to have to find a solution for this. 

cc @garethbowen

Reply:
What's the downside of putting them in the root?

Reply:
they don't get hoisted in the docker image :(

Reply:
Damn, true.

We need to find a proper solution, but to unblock @samuelimoisili can we inline the dependency in the sentinel package.json?

Reply:
@samuelimoisili do you think you can try @garethbowen 's suggestion?